cypress/e2e/euronews/tests.cy.js

```diff
@@ -0,0 +1,135 @@
+/// <reference types="Cypress" />
+import newsletters from '../../fixtures/newsletters.json';
+
+describe('Euronews tests', () => {
+    beforeEach(() => {
+        // Step 1
+        // Follow the euronews.com 
+        cy.visit('https://www.euronews.com/');
```

Comment:
In cypress settings you can define the baseUrl but in that case you work on two domains.
So it should be moved to fixture file with urls.